src/general/requiredlabels/src.cel

```diff
@@ -0,0 +1,5 @@
+validations:
+- expression: '[object, oldObject].exists(obj, obj != null && has(obj.metadata) && variables.params.labels.all(entry, has(obj.metadata.labels) && entry.key in obj.metadata.labels))'
```

Comment:
I think we want `.all()` instead of `.exists()`, otherwise oldObject being compliant will short-circuit throwing a violation for object being non-compliant.

Also, we should not check oldObject at all. That's not how the Rego is written.

Reply:
I tested this in my cluster, `.all` would expect both to exist, which would block good resources on create.

Reply:
Suggestion of removing oldObject SGTM